lib/cloud/aws/policy.go

```diff
@@ -404,13 +400,23 @@ type policies struct {
 	partitionID string
 	// accountID current AWS account ID.
 	accountID string
-	// iamClient already initialized IAM client.
-	iamClient iamiface.IAMAPI
+	// iamClient is an already initialized IAM client.
+	iamClient iamClient
+}
+
+type iamClient interface {
```

Comment:
I'd export this. Otherwise you have an exported constructor func that requires as input an unexported interface type.